packages/forms/src/elements/file-list/components/Close.spec.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render } from 'garden-test-utils';
+import { File } from '../../..';
+
+describe('File.Close', () => {
+  it('renders the expected HTML element', () => {
+    const { container } = render(<File.Close />);
+
+    expect(container.firstChild!.nodeName).toBe('DIV');
+  });
+
+  it('passes ref to underlying DOM element', () => {
+    const ref = React.createRef<HTMLDivElement>();
+    const { container } = render(<File.Close ref={ref} />);
+
+    expect(container.firstChild).toBe(ref.current);
+  });
+
+  it('composes mousedown event', () => {
```

Comment:
Not entirely sure I understand the description of the test, I feel like it's a bit vague. My understanding for the test through the code, you're ensuring that the event only runs once per click. 

I am also not sure if you're indirectly testing for `composeEventHandlers`.?

Reply:
The description is accurate. Emphasizing handler could make it more clear though: "composes mousedown event _handler_". A common way to test this is to use jest mock functions:

```jsx
const onMouseDown = jest.fn();

expect(onMouseDown).toHaveBeenCalledWith(arg);
expect(onMouseDown).toHaveBeenCalledTimes(1);
```

Reply:
I am ensuring that a consumer-provided mouse down event also runs (composed) in conjunction with the internal mouse down handler.